routes/campgrounds.js

```diff
@@ -29,6 +30,17 @@ router.get('/', (req, res) => {
 
 //CREATE ROUTE
 router.post('/', isLoggedIn, (req,res) => {
+    if(
```

Comment:
I guess it would be nice if you could somehow define schemas in general, that would validate the packets incoming.
Just a suggestion

Reply:
Please export it to another function, and put the function invocation inside the if condition